tests/system/action/user/test_update.py

```diff
@@ -128,6 +146,391 @@ def test_update_temporary_user_error(self) -> None:
+            {
+                "id": 111,
+                "username": "new username",
+                "title": "new title",
+                "first_name": "new first_name",
+                "last_name": "new last_name",
+                "is_active": True,
+                "is_physical_person": True,
+                "default_password": "new default_password",
+                "gender": "female",
+                "email": "info@openslides.com ",
+                "default_number": "new default_number",
+                "default_structure_level": "new default_structure_level",
+                "default_vote_weight": "1.234000",
+                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_USERS,
+                "committee_as_member_ids": [60],
+                "committee_as_manager_ids": [63],
+                "guest_meeting_ids": [1, 4],
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/111",
+            {
+                "username": "new username",
+                "title": "new title",
+                "first_name": "new first_name",
+                "last_name": "new last_name",
+                "is_active": True,
+                "is_physical_person": True,
+                "default_password": "new default_password",
+                "gender": "female",
+                "email": "info@openslides.com ",
+                "default_number": "new default_number",
+                "default_structure_level": "new default_structure_level",
+                "default_vote_weight": "1.234000",
+                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_USERS,
+                "committee_as_member_ids": [60],
+                "committee_as_manager_ids": [63],
+                "guest_meeting_ids": [1, 4],
+            },
+        )
+
+    def test_update_permission_manage_user_no_permission(self) -> None:
+        """ May update group A fields only """
+        self.permission_setup()
+        self.create_meeting(base=4)
+        self.set_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+
+        response = self.request(
+            "user.update",
+            {
+                "id": 111,
+                "username": "new username",
+                "structure_level_$": {"1": "group B field"},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "You are not allowed to perform action user.update. Missing permissions {'_User.CAN_MANAGE for meeting 1'}",
+            response.json["message"],
+        )
+
+    def test_update_permission_user_can_manage(self) -> None:
+        """ May update group B and C fields """
+        self.permission_setup()
+        self.create_meeting(base=4)
+        self.set_management_level(None, self.user_id)
+        self.set_user_groups(self.user_id, [3, 6])
```

Comment:
what are those groups?

Reply:
These are the empty groups of meeting/1 and meeting/4. Comment in src

Reply:
Done